down-the-stack/solutions/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+authors = ["Tanks Transfeld <tanks.transfeld@ferrous-systems.com>"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+name = "uarte_enable_solution"
```

Comment:
This `Cargo.toml` contains solutions to all the apps (which are binary crates within this package), so I think it is better named "solutions".